app/src/js/mousePosition.js

```diff
@@ -21,7 +21,8 @@ export default L.Control.MousePosition = L.Control.extend({
     separator: ", ",
     numDigits: 5,
     prefix: "",
-    targetPlanet: ""
+    targetPlanet: "",
+    lngFirst: true
```

Comment:
I think the map by default needs to show lat first, unless we swapped the coordinates somewhere. Lat should be between -90, 90

Reply:
Nevermind!